js/pulse.js

```diff
@@ -359,10 +359,14 @@
 
                 if(!this._pulseHost) {
                     log('No Pulse hostname found in plugin parameters or media metadata; will not attempt to show Pulse ads');
+                    this.ready = true;
+                    amc.onAdManagerReady();
                     return;
                 }
 
                 this.ready = true;
+                amc.onAdManagerReady();
```

Comment:
hmm, I'm not sure honestly, does it mean `INITIAL_PLAY_REQUESTED` can be triggered? I think it should be fine